src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/MapAction_ExplicitBodyParam_ComplexReturn_Snapshot.generated.txt

```diff
@@ -283,7 +295,7 @@ namespace Microsoft.AspNetCore.Http.Generated
         [UnconditionalSuppressMessage("Trimming", "IL2026:RequiresUnreferencedCode",
             Justification = "The 'JsonSerializer.IsReflectionEnabledByDefault' feature switch, which is set to false by default for trimmed ASP.NET apps, ensures the JsonSerializer doesn't use Reflection.")]
         [UnconditionalSuppressMessage("AOT", "IL3050:RequiresDynamicCode", Justification = "See above.")]
```

Comment:
```suggestion
```

These should no longer be needed. You only need suppressions on the method calling the "unsafe" method. This method now calls `WriteJsonResponseAsync` which isn't annotated as "unsafe".